examples/DancingGoat/Program.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Threading.Tasks;
+
+using DancingGoat;
+using DancingGoat.Models;
+
+using Kentico.Activities.Web.Mvc;
+using Kentico.Content.Web.Mvc.Routing;
+using Kentico.Membership;
+using Kentico.OnlineMarketing.Web.Mvc;
+using Kentico.PageBuilder.Web.Mvc;
+using Kentico.Web.Mvc;
+
+using Kentico.Xperience.Cloud;
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Mvc.Routing;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+
+using Microsoft.Extensions.Hosting;
+using DancingGoat.Search;
+
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddXperienceCloudApplicationInsights(builder.Configuration);
+
+if (builder.Environment.IsQa() || builder.Environment.IsUat() || builder.Environment.IsProduction())
+{
+    builder.Services.AddKenticoCloud(builder.Configuration);
+    builder.Services.AddXperienceCloudSendGrid(builder.Configuration);
+}
+
+builder.Services.AddKentico(features =>
+{
+    features.UsePageBuilder(new PageBuilderOptions
+    {
+        DefaultSectionIdentifier = ComponentIdentifiers.SINGLE_COLUMN_SECTION,
+        RegisterDefaultSection = false,
+        ContentTypeNames = new[]
+        {
+            LandingPage.CONTENT_TYPE_NAME,
+            ContactsPage.CONTENT_TYPE_NAME,
+            ArticlePage.CONTENT_TYPE_NAME
+        }
+    });
+
+    features.UseWebPageRouting();
+    features.UseEmailMarketing();
+    features.UseEmailStatisticsLogging();
+    features.UseActivityTracking();
+});
+
+builder.Services.Configure<RouteOptions>(options => options.LowercaseUrls = true);
+
+builder.Services.AddLocalization()
+    .AddControllersWithViews()
+    .AddViewLocalization()
+    .AddDataAnnotationsLocalization(options =>
+    {
+        options.DataAnnotationLocalizerProvider = (type, factory) => factory.Create(typeof(SharedResources));
+    });
+
+builder.Services.AddDancingGoatServices();
+
+builder.Services.AddKenticoAlgoliaServices(builder.Configuration);
```

Comment:
This will need updated to use the new extension.

Reply:
What do you mean ? This extension method uses services.AddKenticoAlgolia() extension method.

Reply:
Ahh! I didn't realize this was an extension in `DancingGoat`. Makes sense!